requirements/docs.txt

```diff
@@ -3,10 +3,8 @@
 
 # Make sure to run `make docs` when this is updated to verify the build.
 Sphinx==3.3.1
-
-# FIXME Pointing at a personal branch with specific changes.
-#       These changes have been submitted upstream (unaguil/sphinx-swaggerdoc).
-##sphinx-swaggerdoc[yaml]
--e git+https://github.com/pumazi/sphinx-swaggerdoc.git@for-use#egg=sphinxcontrib-swaggerdoc
 PyYAML==5.3.1
+
+# FIXME Pointing at a forked branch with specific changes.
+-e git+https://github.com/openstax/sphinx-swaggerdoc.git@for-use#egg=sphinx-swaggerdoc
```

Comment:
this needs to be `sphinxcontrib-swaggerdoc` in the egg argument (new pip complains about this)
or this LOC needs to be changed in the repo pointing to it: https://github.com/openstax/sphinx-swaggerdoc/blob/05c6bfeb3ea562dccd6105cef8402665917365a1/setup.py#L13

Reply:
This should be resolved now.